metricbeat/module/kubernetes/vendor/vendor.json

```diff
@@ -0,0 +1,43 @@
+{
+	"comment": "",
+	"ignore": "test",
```

Comment:
you need to add `github.com/elastic/beats` here as otherwise when you remove all the beats dependencies will be added to when doing an update.